README.md

```diff
@@ -45,7 +45,7 @@ Edit the command however you need, and paste it into your CI pipeline.
 
 ## Supported targets
 
-Check [supported-targets](/supported-targets) for lists of targets quickinstall
+Check [supported-targets](/cronjob_scripts/trigger-package-build.py) for lists of targets quickinstall
```

Comment:
Can we keep the `supported-targets`?

cargo-binstall uses it to avoid hitting github APIs, if binstall knows that quickinstall does not support the target.